lang/stack/Portfile

```diff
@@ -43,13 +43,13 @@ distfiles           ${distname}-bin:release \
 extract.only        ${github.tag_prefix}${github.version}${extract.suffix}
 
 checksums           ${distname}-bin \
-                    rmd160  01fb6c671882d667611b247b1d0ec147f09f1405 \
-                    sha256  dd31f7827172f07e3f6bd0145bd6f1e67fb9ce88da0bf6103ce0f13b25a8b6e3 \
-                    size    36107528 \
+                    rmd160  fbe0175e8fb6f2c328aecb2052223871dc5ca74b \
+                    sha256  8efba112e7cb3cabbe96bad7c70b8a1c3ed63bb1f0fa50815ac7d5e60e794eb8 \
+                    size    36142208 \
                     ${github.tag_prefix}${github.version}${extract.suffix} \
-                    rmd160  bc30354c95c82d29ff124bfd23b2270086a95f3c \
-                    sha256  eb849d5625084a6de57e8520ddf8172aca64ddadd9fee37cdafeefad80895b62 \
-                    size    686145 \
+                    rmd160  0d38bb22f8a980f67ccef926713912ded02acbad \
+                    sha256  37f4bc0177534782609ec3a67ec413548d3f2cabff7c4c0bc8a92a36e49c6877 \
+                    size    689876 \
                     ${distname}-bin.asc \
```

Comment:
Should we add additional recommended checksums for this `.asc` file?

```
Warning: stack-2.7.3-osx-x86_64-bin.asc - missing recommended checksum type: rmd160
Warning: stack-2.7.3-osx-x86_64-bin.asc - missing recommended checksum type: sha256
```

Reply:
That’s not necessary—it’d just be checksumming a checksum. This lint error is a false alert.